avian2d/src/lib.rs

```diff
@@ -132,6 +144,11 @@ fn update_proximity_sensors_system(
                 TnuaToggle::SenseOnly => {}
                 TnuaToggle::Enabled => {}
             }
+            let transform = Transform {
```

Comment:
I think this `transform` is only used for computing the `cast_origin`. Maybe we can just compute it directly with the rotation and position instead of creating a transform matrix?

Reply:
I don't know how we would do that. Creating the transform matrix is super cheap, so I also don't think this is pressing.